lldb/source/Plugins/LanguageRuntime/Swift/SwiftLanguageRuntime.cpp

```diff
@@ -2586,7 +2586,7 @@ lldb::addr_t SwiftLanguageRuntime::GetAsyncContext(RegisterContext *regctx) {
 /// operation. This is only valid for x86_64 or aarch64.
 llvm::ArrayRef<uint8_t>
 GetAsyncRegFromFramePointerDWARFExpr(llvm::Triple::ArchType triple,
-                                     bool with_deref) {
+                                     bool with_double_deref) {
```

Comment:
Update function docs

Reply:
TODO for much later: this should be a virtual method in some target plugin.